README.md

```diff
@@ -24,24 +24,23 @@ Much of the code is licensed under the Apache 2.0 license. See the [LICENSE](LIC
 We recommend the following hardware to run *stretch-ai*. Other GPUs and other versions of Stretch may support some of the capabilities found in this repository, but our development and testing have focused on the following hardware.
 
 - **[Stretch 3](https://hello-robot.com/stretch-3-product) from [Hello Robot](https://hello-robot.com/)**
-  - When *Checking Software*, `stretch_system_check.py` should report that all software passes.
-  - You can find detailed instructions in the [official guide for updating Stretch 3's software](https://docs.hello-robot.com/0.3/software/updating_software/).
```

Comment:
Why is this deleted

Reply:
Robot-side software is installed in a docker image anyways. We only care that the Checking Hardware section reports all green.

Reply:
See the green part:
![Screenshot from 2024-11-14 00-12-39](https://github.com/user-attachments/assets/6e7af55c-6dd0-475c-9171-3e51feb331d1)